Apache-Arrow-Flight-Manager-Tester/main.py

```diff
@@ -0,0 +1,116 @@
+import pprint
+from typing import Literal
+
+from pyarrow import flight, Schema
+from pyarrow._flight import FlightInfo, FlightClient, ActionType, FlightEndpoint
+
+
+# PyArrow Functions.
+def list_flights(flight_client: FlightClient) -> list[FlightInfo]:
+    response = flight_client.list_flights()
+
+    return list(response)
+
+
+def get_schema(flight_client: FlightClient, table_name: str) -> Schema:
+    upload_descriptor = flight.FlightDescriptor.for_path(table_name)
+    response = flight_client.get_schema(upload_descriptor)
+
+    return response.schema
+
+
+def do_action(flight_client: FlightClient, action_type: str, action_body: str) -> list[any]:
```

Comment:
Can the return type be made more specific? Looks like [`do_action()`](https://arrow.apache.org/docs/python/generated/pyarrow.flight.FlightClient.html#pyarrow.flight.FlightClient.do_action) always return the same [`Result`](https://arrow.apache.org/docs/python/generated/pyarrow.flight.Result.html) type.

Reply:
Fixed.